### ISSUE TITLE
Documentation Error for Response parameters

### ISSUE THREAD

**Issue by frederikaalund** [aio-libs/aiohttp#944]
Currently, the [documentation](http://aiohttp.readthedocs.io/en/stable/web_reference.html#aiohttp.web.Response) for

```
class aiohttp.web.Response(*, status=200, headers=None, content_type=None, charset=None, body=None, text=None)
```

states that

> content_type (str) – response’s content type. 'text/plain' if text is passed also, 'application/octet-stream' otherwise.

However, if I do not set the `text` argument (but set the `body` argument instead) the `content_type` is not set at all (whereas the documentation says it should be automatically set to `application/octet-stream`).

Python 3.5.1 and aiohttp 0.21.0

**Comment by avamsi** [comment]
@asvetlov What is the intended behavior in this case?
I mean, is this a documentation error or implementation error?

**Comment by asvetlov** [comment]
It's implementation bug.
Would you create  a  Pull Request for fixing it?
Default `content-type` for prepared response should be `application/octet-stream` if the content type was not specified.

**Comment by avamsi** [comment]
Sure thing

**Comment by asvetlov** [comment]
Fixed by #1124
